examples/heartrate_monitor_peripheral.py

```diff
@@ -0,0 +1,117 @@
+"""Example of how to create a Peripheral device/GATT Server"""
+# Standard modules
+import logging
+import struct
+
+# Bluezero modules
+from bluezero import async_tools
+from bluezero import adapter
+from bluezero import peripheral
+
+HRM_SRV = '0000180D-0000-1000-8000-00805f9b34fb'
+HR_MSRMT_UUID =      '00002a37-0000-1000-8000-00805f9b34fb'
+BODY_SNSR_LOC_UUID = '00002a38-0000-1000-8000-00805f9b34fb'
+HR_CTRL_PT_UUID =    '00002a39-0000-1000-8000-00805f9b34fb'
+
+
+heartrate = 60
+def read_heartrate():
+    global heartrate
+    """
+    Example read callback. Value returned needs to a list of bytes/integers
+    in little endian format.
+
+    // Send one byte of flags
+    // As well as an 8 bit measurement
+
+    :return: list of uint8 values
+    """
+    flags = 0b00000110
+    heartrate = heartrate + 1
+    print("Sending heartrate value of %d" % (heartrate))
+    if heartrate > 180:
+        heart_rate = 60
+    return struct.pack('<BB', flags, heartrate)
+
+def read_sensor_location():
```

Comment:
Add a doc string for this method.
I think it would also be good to either document all the options or reference the documentation on bluetooth.com where people can find all the options